packages/server/src/sync/schema.ts

```diff
@@ -16,7 +16,7 @@ export const makeSyncRequestSchema = (channels: Channel[]) => {
     channels: Joi.object(channelsSchema).allow(null),
     webhooks: Joi.array().items(SyncWebhookSchema).allow(null),
     id: Joi.string().guid().allow(null),
-    token: Joi.string().length(88).allow(null),
+    token: Joi.string().length(125).allow(null),
```

Comment:
We should probably export those values by default so we can reuse them in our tests. What do you think?